### PR TITLE
Prepare next release

### PR DESCRIPTION
**Pull request by jderusse**

### CODE REVIEW

File: src/Core/CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,20 @@
...
 
+## 1.9.0
+
 ### Added
 
 - Changed case of object's properties to camelCase.
 - Added documentation in class's headers.
 - Removed `final` from `ClientException` and `ServerException`.
 - Make Responses thrown Business Exception when AwsErrorCode <-> Exception class mapping provided through RequestContext.
 - Added Business Exceptions.
```
Comment (Nyholm) [inline src/Core/CHANGELOG.md:13]: Do you mean "domain exceptions"?

File: src/Core/CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,20 @@
 
 ## NOT RELEASED
 
+## 1.9.0
+
 ### Added
 
 - Changed case of object's properties to camelCase.
 - Added documentation in class's headers.
```
Comment (Nyholm) [inline src/Core/CHANGELOG.md:10]: I think one can say "class headers".

"class's headers" is the "headers owned by a class".

**Review by Nyholm — APPROVED** [review]
Just some minors, sorry for not finding these earlier. I know that these changes apply for most CHANGELOG files.

File: src/Core/CHANGELOG.md (hunk 1 above)
Comment (jderusse) [reply src/Core/CHANGELOG.md]: yeah. I was thinking about business logic...

File: src/Core/CHANGELOG.md (hunk 2 above)
Comment (jderusse) [reply src/Core/CHANGELOG.md]: sounds better thank you

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**